system-variables.md

```diff
@@ -1566,73 +1559,63 @@ SET tidb_slow_log_threshold = 200;
 - Default value: ""
 - This variable is used to set the time point at which the data is read by the session. For example, when you set the variable to "2017-11-11 20:20:20" or a TSO number like "400036290571534337", the current session reads the data of this moment.
 
-### tidb_stats_load_sync_wait <span class="version-mark">New in v5.4.0</span>
+### tidb_stats_load_pseudo_timeout <span class="version-mark">New in v5.4.0</span>
 
-> **WARNING:**
+> **Warning:**
```

Comment:
These two variables were in the wrong alphabetical order. Warning is also in upper case (inconsistent).